Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -69,6 +72,9 @@ ProjectData::ProjectData(ConfigTree const& project_config,
 
 	// output
 	parseOutput(project_config.get_child("output"), path);
+
+	// timestepping
+	parseTimeStepping(project_config.get_child("time_stepping"));
```

Comment:
I think it is better to explicitly state the time-stepping scheme in the config files; this can reduce number of surprised users.